### ISSUE TITLE
Warning: Accessing PropTypes

### ISSUE THREAD

**Issue by epotockiy** [springload/react-accessible-accordion#9]
I get a warning

```javascript
Warning: Accessing PropTypes via the main React package is deprecated. Use the prop-types package from npm instead.
```

I checked all my components and figured out that error occurred after adding this accordion component.

Could you fix this?

**Comment by thibaudcolas** [comment]
Thanks for the report @epotockiy, this is because this component hasn't been migrated to support React 15.5 and above yet.

We will get to it when we need it (the component still works the same, it's just a warning), if you need this soon could you make a pull request with the upgrade? I'm more than happy to help you along the way if you're up for it.

**Comment by vincentaudebert** [comment]
@epotockiy This is now fixed in `0.4.0`. Thanks :)
